addons/sourcemod/scripting/hextags.sp

```diff
@@ -303,6 +312,49 @@ public void warden_OnWardenRemoved(int client)
 
 }
 
+public Action Cmd_Anonymous(int client, int args)
+{
+	if (AreClientCookiesCached(client))
+	{
+		char sCookieValue[12];
+		GetClientCookie(client, hVibilityAdminsCookie, sCookieValue, sizeof(sCookieValue));
+		int cookieValue = StringToInt(sCookieValue);
+		if (cookieValue == 1)
+		{
+			cookieValue = 0;
+			g_hAnonymous[client] = false;
+			IntToString(cookieValue, sCookieValue, sizeof(sCookieValue));
+			SetClientCookie(client, hVibilityAdminsCookie, sCookieValue);
+			LoadTags(client);
+			char sTag[32];
+			CS_GetClientClanTag(client, sTag, sizeof(sTag));
+			ReplyToCommand(client, "[SM] You are no longer anonymous. Your tag is %s", selectedTags[client].ScoreTag);
+			if (StrEqual(sTag, selectedTags[client].ScoreTag))
+			{
+				return Plugin_Handled;
+			}
+			CS_SetClientClanTag(client, selectedTags[client].ScoreTag);
+		}
+		else
+		{
+			cookieValue = 1;
+			g_hAnonymous[client] = true;
+			IntToString(cookieValue, sCookieValue, sizeof(sCookieValue));
+			SetClientCookie(client, hVibilityAdminsCookie, sCookieValue);
+			LoadTags(client);
+			char sTag[32];
+			CS_GetClientClanTag(client, sTag, sizeof(sTag));
+			ReplyToCommand(client, "[SM] You are now anonymous. Your tag is %s", selectedTags[client].ScoreTag);
+			if (StrEqual(sTag, selectedTags[client].ScoreTag))
+			{
+				return Plugin_Handled;
+			}
+			CS_SetClientClanTag(client, selectedTags[client].ScoreTag);
+		}
```

Comment:
The code in the two if branches is pretty much the same except for the Chat  messages so you could just to smth like:
```c
cookieValue = !cookieValue;
g_hAnonymous[client] = cookieValue == 1;
IntToString(cookieValue, sCookieValue, sizeof(sCookieValue));
SetClientCookie(client, hVibilityAdminsCookie, sCookieValue);
LoadTags(client);
char sTag[32];
CS_GetClientClanTag(client, sTag, sizeof(sTag));
if (!cookieValue)
{
	ReplyToCommand(client, "[SM] You are no longer anonymous. Your tag is %s", selectedTags[client].ScoreTag);
} 
else 
{
	ReplyToCommand(client, "[SM] You are now anonymous. Your tag is %s", selectedTags[client].ScoreTag);
}		
if (StrEqual(sTag, selectedTags[client].ScoreTag))
{
	return Plugin_Handled;
}
CS_SetClientClanTag(client, selectedTags[client].ScoreTag);
```